docs/docusaurus/docs/guides/expectations/how_to_edit_an_existing_expectationsuite.md

```diff
@@ -102,6 +102,8 @@ It runs the `expect_column_values_to_be_between` Expectation on the `passenger_c
 
 Here is the same configuration, but this time as a `ExpectationConfiguration` object.  
 
+First run `from great_expectations.core.expectation_configuration import ExpectationConfiguration`
+
```

Comment:
Since this is a line of code we want to see in use, we should be pulling it from an example script under test.  I checked, and that line is in the example script.  We can pair when you're available and I'll walk you through converting that to a snippet that you can pull into a Python code block.